dojo_app.py

```diff
@@ -0,0 +1,115 @@
+"""
+This Application automatically allocated spaces to people at random
+Usage:
+    dojo_app.py create_room <room_type> <room_name>...
+    dojo_app.py add_person <first_name> <last_name> <FELLOW/STAFF> [<wants_accomodation>]
+    dojo_app.py print_room <room_name>
+    dojo_app.py print_allocations [--o=filename]
+    dojo_app.py print_unallocated [--o=filename]
+    dojo_app.py reallocate_person <person_identifier> <new_room_name>​
+    dojo_app.py load_people <text_file>
+    dojo_app.py -h | --help
+    dojo_app.py -V | --version
+    dojo_app.py -i | --interactive
+
+Options:
+    -o                      Outputs the result into a text file.
+    -h, --help              Show message and exit.
+    -V, --version           Show the version.
+    -i, --interactive       interactive.
+
+
+"""
+import sys
+import cmd
```

Comment:
I would suggest you adhere to the [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) import order here